src/main/resources/database/changes/release-10.59.0/add-H-type-to-case-categories.sql

```diff
@@ -0,0 +1,19 @@
+UPDATE casesvc.category
+SET oldcasesampleunittypes='BI,H'
+WHERE categorypk='SUCCESSFUL_RESPONSE_UPLOAD';
+
+UPDATE casesvc.category
+SET oldcasesampleunittypes='B,H'
+WHERE categorypk='ACCESS_CODE_AUTHENTICATION_ATTEMPT';
+
+UPDATE casesvc.category
```

Comment:
This might seem a bit picky but IN syntax is much cleaner:
UPDATE casesvc.category
SET oldcasesampleunittypes='B,BI,H'
WHERE categorypk IN ('ACTION_CANCELLATION_CREATED','COMPLETED_BY_PHONE','NO_LONGER_REQUIRED','ACTION_CANCELLATION_COMPLETED','EQ_LAUNCH','CASE_CREATED','ACTION_COMPLETED','ACTION_UPDATED','ACTION_CREATED');